draft-ietf-teep-protocol.md

```diff
@@ -1512,6 +1500,26 @@ Compromised Time Source
   source of time, such as {{RFC8915}}.  A compromised time source could
   thus be used to subvert such validity checks.
 
+# Privacy Considerations {#privacy}
+
+Depending on
+the properties of the attestation mechanism, it is possible to
+uniquely identify a device based on information in the
+attestation payload or in the certificate used to sign the
+attestation payload.  This uniqueness may raise privacy concerns. To lower the
+privacy implications the TEEP Agent MUST present its
+attestation payload only to an authenticated and authorized TAM and when using
+EATS, it SHOULD use encryption as discussed in {{I-D.ietf-rats-eat}}, since
+confidentiality is not provided by the TEEP protocol itself and
+the transport protocol under the TEEP protocol might be implemented
+outside of any TEE. If any mechanism other than EATs is used, it is
```

Comment:
I'm less and less convinced we should pluralize EAT.

Reply:
> I'm less and less convinced we should pluralize EAT.

EAT = Entity Attestation Token.  When referring to messages in particular, I think it should be plural when one could replace it with "tokens" and be grammatically correct.

Reply:
Updated